lib/mutant/runner.rb

```diff
@@ -33,6 +33,12 @@ def initialize(*)
     def run_mutation_analysis
       @result = run_driver(Parallel.async(mutation_test_config))
       reporter.report(@result)
+
+      if env.config.json_dump
```

Comment:
To make my point clear, the call to `reporter.report(@result)` just above would cover your use case: https://github.com/jheth/mutant/blob/eab3de25ead132e5d876c09b8570260dbc40c61b/lib/mutant/runner.rb#L35